src/AudioToolbox/AudioType.cs

```diff
@@ -451,7 +455,10 @@ public enum AudioChannelLabel : int { // UInt32 AudioChannelLabel
 		HoaAcn13               = (2 << 16) | 13,
 		HoaAcn14               = (2 << 16) | 14,
 		HoaAcn15               = (2 << 16) | 15,
-		HoaAcn65024            = (2 << 16) | 65024  
+		HoaAcn65024            = (2 << 16) | 65024,
+
+		BeginReserved          = unchecked((int)0xF0000000),
+		EndReserved            = unchecked((int)0xFFFFFFFE),
```

Comment:
I'm not a fan of sentinel values in _normal_ enums. They could change over time (breaking change for us) and they are hard to use/understand for developers as the `enum` is not decorated with a `[Flag]` attribute.

Now it can be useful :) so what about using a extension method, just like we do for smart enums, so people can do:

```
var x = GetEnumValue ().IsReversed ();
```

That would hide the values (in case they change) and be easier to use.

Reply:
🤣 I meant `IsReserved` 🤣

Reply:
Agree with @spouliot, using an extension method to do this sounds like a great idea